server/graphql/mutations.js

```diff
@@ -379,13 +380,13 @@ const mutations = {
     },
     resolve: async (_, args, req) => createPaymentMethod(args, req.remoteUser),
   },
-  claimVirtualCard: {
+  claimPaymentMethod: {
```

Comment:
Does it make sense creating a "more generic" name in a case where only virtual cards can be claimed today? I know we can have other kinds of payment methods later on that can be claimed, but this may sound confusing if we consider the other payment methods that cannot be claimed. not a big deal though.

Reply:
I see what you mean but I'd rather limit the number of those graphql mutations and therefore keep them as generic as possible. Less surface to test / maintain.